crates/daphne/src/messages/mod.rs

```diff
@@ -760,7 +760,7 @@ impl TryFrom<u8> for TransitionFailureDraft09 {
     }
 }
 
-impl TryFrom<u8> for TransitionFailureLatest {
+impl TryFrom<u8> for ReportErrorLatest {
```

Comment:
```suggestion
impl TryFrom<u8> for ReportError {
```

Reply:
So the unified enum is called `ReportError` which I think makes the most sense. We could rename the unified enum to `UnifiedReportError` but I think that makes the code less readable. Alternatively I could rename `ReportErrorLatest` to `ReportErrorDraft13` (or maybe `ReportErrorDraft12` as that was when the renumbering happened) just so that it will stay aligned with the drafts.

Reply:
Nope, better as is!